src/utils/verifyEDLJwt.js

```diff
@@ -12,13 +12,14 @@ const jwt = require('jsonwebtoken')
 export const verifyEDLJwt = async (header) => {
   // Edl token is in the form of Bearer <Token> retrieve <token>
   const kid = process.env.edlKeyId
+
   const bearerHeader = header.split(' ')
   const [, token] = bearerHeader
 
   // Retrieve all of the keysValue pairs from the JWKS env var return jwksClient obj
   const client = jwksClient({
     getKeysInterceptor: () => {
-      const rsaEnv = JSON.parse(process.env.rsaKey)
+      const rsaEnv = JSON.parse(process.env.edlJwk)
```

Comment:
Does this variable name make more sense if it is called `edlJwk` instead of `rsaEnv`? Just pointing out the inconsistency in names